romio.spec

```diff
@@ -25,7 +24,7 @@ URL:        http://www.mpich.org/
 %endif
 %endif
 %endif
-Source0:    https://build.hpdd.intel.com/job/daos-stack/job/mpich/job/daos_adio-rpm/lastSuccessfulBuild/artifact/artifacts/%{distro}/%{name}-%{version}.tar.gz
+Source0:    https://build.hpdd.intel.com/job/daos-stack/job/mpich/job/PR-44/lastSuccessfulBuild/artifact/artifacts/%{distro}/%{name}-%{version}.tar.gz
```

Comment:
This is not good.  This should be (only) optionally overridden by defining a macro externally to represent the value we want, and falling back to `daos_adio-rpm` if the caller doesn't set the macro value.

Our `packaging/` code has the ability to do that.  We do it in other repos for defining a git hash in the `Release:` tag using a `%{relval}` macro, for example.  The only thing we don't do with `%{relval}` that you'd want here is falling back to a value if it's not defined externally.

Just going to -1 for this to make sure this doesn't slip through.  We really don't want to land this with `PR-44` in this path.

Reply:
Perhaps after mpich PR-44 has been landed first, this PR could be modified to revert the change of this Source0 setting? We have not had the chance to implement the override machinery for this purpose.

Reply:
Indeed.

Reply:
This will need to be fixed once daos-stack/mpich#44 lands before landing this one.  There is no need to land this and then fix it up with a subsequent landing.